frontend/src/shared/helpers.tsx

```diff
@@ -224,7 +224,11 @@ export const SampleMetadataDetailsColumns: ColDef[] = [
     headerName: "Status",
     cellRenderer: (params: ICellRendererParams) => {
       if (params.data?.revisable) {
-        return params.data?.validationStatus ? <CheckIcon /> : <WarningIcon />;
```

Comment:
`validationStatus` can be either `true`, `false`, or `null`. There is a validation report to show only when `validationStatus === false`. This old code incorrectly renders a warning icon when `validationStatus === null`.